pkg/testcase/ingressdns.go

```diff
@@ -83,8 +83,8 @@ func TestIngressRoute(applyWorkload, deleteWorkload bool, apiVersion string) {
 			Expect(err).NotTo(HaveOccurred(), "failed to read file for ingressroute resource")
 		}
 
-		publicIp := fmt.Sprintf("%s.nip.io", shared.FetchNodeExternalIP()[0])
-		replacer := strings.NewReplacer("$YOURDNS", publicIp, "$APIVERSION", apiVersion)
+		pIp := fmt.Sprintf("%s.nip.io", shared.FetchNodeExternalIP()[0])
```

Comment:
NIT: curious.. why are we changing this var name to pIp from publicIp - publicIp feels like a better descriptive name?

Reply:
hey sorry to merge that if you still had questions

so basically here https://github.com/fmoral2/distros-test-framework/blob/a2dddd8f85a09677493014bc0bf33ba272dad74f/pkg/testcase/ingressdns.go#L74

its already a var with the same name, to no create a shadow or we need to rename or create something similar to a closure , which in the context would be an overkill